assets/css/theme-tugboat.css

```diff
@@ -0,0 +1,438 @@
+    
+    /* Headings use Poppins */
+    --MAIN-TITLES-font: "Poppins", sans-serif;
+    --MAIN-TITLES-font-weight: 500; /* Medium weight for headings */
+    --MAIN-TITLES-H1-font: "Poppins", sans-serif;
+    --MAIN-TITLES-H1-font-weight: 900; /* Bold for H1 */
+    --MAIN-TITLES-H1-letter-spacing: -0.03em;
+    --MAIN-TITLES-H1-text-transform: none;
+    --MAIN-TITLES-H2-font: "Poppins", sans-serif;
+    --MAIN-TITLES-H2-font-weight: 700; /* Bold weight for H2 */
+    --MAIN-TITLES-H2-letter-spacing: -0.02em;
+
+    /* Chroma theme selection */
+    --CODE-theme: relearn-light; /* Light syntax highlighting by default */
+}
+
+/* Light mode colors */
+@media (prefers-color-scheme: light) {
+    :root {
+        --PRIMARY-color: #0098CB;
+        --SECONDARY-color: #07688C;
+        --ACCENT-color: #03384B;
+
+        --MAIN-TEXT-color: #1C4248; /* Color of text by default */
+        --MAIN-TITLES-TEXT-color: #1C4248; /* Color of titles h2-h3-h4-h5 */
+        --MAIN-LINK-color: #0098CB; /* Color of links */
+        --MAIN-LINK-HOVER-color: #167ad0; /* Color of hovered links */
+        --MAIN-ANCHOR-color: #1C90F3; /* color of anchors on titles */
+        --MAIN-BG-color: #ffffff; /* Main content background */
+
+        --MENU-HEADER-BG-color: #E8F0F5; /* Background color of menu header */
+        --MENU-HEADER-BORDER-color: #E8F0F5; /*Color of menu header border */
+
+        --MENU-SEARCH-BG-color: #E8F0F5; /* Search field background color (by default borders + icons) */
+        --MENU-SEARCH-BG-color: #F9FCFE; /* Search field background color (by default borders + icons) */
+        --MENU-SEARCH-BOX-color: #E8F0F5; /* Override search field border color */
+        --MENU-SEARCH-BOX-ICONS-color: #1C4248; /* Override search field icons color */
+
+        --MENU-SECTIONS-ACTIVE-BG-color: #fff; /* Background color of the active section and its childs */
+        --MENU-SECTIONS-BG-color: #E8F0F5; /* Background color of other sections */
+        --MENU-SECTIONS-LINK-color: #535B60; /* Color of links in menu */
+        --MENU-SECTIONS-LINK-HOVER-color: #07688C;  /* Color of links in menu, when hovered */
+        --MENU-SECTION-ACTIVE-CATEGORY-color: #03384B; /* Color of active category text */
+        --MENU-SECTION-ACTIVE-CATEGORY-BG-color: #fff; /* Color of background for the active category (only) */
+        --MENU-SECTION-ACTIVE-CATEGORY-BORDER-color: #E8F0F5;
+
+        --MENU-VISITED-color: #33a1ff; /* Color of 'page visited' icons in menu */
+        --MENU-SECTION-HR-color: #20272b; /* Color of <hr> separator in menu */
+
+        /* Notice/Box colors for light mode */
+        --BOX-CAPTION-color: rgba(255, 255, 255, 1); /* text color of colored box titles */
+        --BOX-BG-color: rgba(255, 255, 255, 0.9); /* background color of colored boxes */
+        --BOX-TEXT-color: rgba(28, 66, 72, 1); /* text color of colored box content */
+        --BOX-GREY-color: rgba(232, 240, 245, 1); /* background color of grey boxes */
+
```

Comment:
![Screenshot 2025-06-13 at 8 01 02 AM](https://github.com/user-attachments/assets/5b74604f-2845-4a2a-9949-ecc832c5a4f9)

I noticed that the callout boxes were quite bright. Could we change them to something like the suggestion below? What do you think?


```suggestion
-- VARIABLE-B0X-color: #0098cb
```
![Screenshot 2025-06-13 at 8 02 21 AM](https://github.com/user-attachments/assets/76f44f67-af27-4f68-b9b6-080becd8b35d)